source/javascripts/site.js

```diff
@@ -1,7 +1,12 @@
 // This is where it all goes :)
 
+//wait for DOM to load
+window.addEventListener('load', (event) => {
+  console.log('page is fully loaded');
```

Comment:
why do we need to console log that the page is loaded? are you using this somewhere? if not please remove

Reply:
Removed